diff --git a/dist/index.esm.js b/dist/index.esm.js
index c97bd358245b9b1566325c56517e9981038c275e..a4dc98a7fdc70f4ab726d4f7bd2db2c3e9ec15ab 100644
--- a/dist/index.esm.js
+++ b/dist/index.esm.js
@@ -1001,7 +1001,7 @@ function BasicChart({ data, setEchartInstance, rowHeight, isMobile = false, hFac
         const colorOpt = ((_d = config.colors) === null || _d === void 0 ? void 0 : _d.length) ? { color: config.colors } : {};
         const aria = {
             enabled: true,
-            decal: { show: true },
+            decal: { show: false },
             label: { enabled: true, description: config.title || config.description || "" },
         };
         let options = Object.assign(Object.assign(Object.assign(Object.assign({}, colorOpt), axis), { grid,
@@ -1096,7 +1096,7 @@ function GeoMapChart({ data, id, setEchartInstance, hFactor = 1, rowHeight, }) {
             },
             aria: {
                 enabled: true,
-                decal: { show: true },
+                decal: { show: false },
                 label: { enabled: true, description: (config === null || config === void 0 ? void 0 : config.title) || (config === null || config === void 0 ? void 0 : config.description) || "" },
             },
             tooltip,
@@ -1282,7 +1282,7 @@ function PieChart({ id, data, setEchartInstance, isMobile = false, rowHeight, hF
                 },
             } }, colorOpt), { aria: {
                 enabled: true,
-                decal: { show: true },
+                decal: { show: false },
                 label: { enabled: true, description: config.title || config.description || "" },
             }, series: Object.assign(Object.assign({}, dataSource.series), { labelLine: {
                     show: showLabels && config.labeLine,
diff --git a/dist/index.js b/dist/index.js
index ab831548c7302236381e7258a5c373773e8e0ec2..bc4cde8fcb97f13dcd81604b6310e03f30f6e85a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1022,7 +1022,7 @@ function BasicChart({ data, setEchartInstance, rowHeight, isMobile = false, hFac
         const colorOpt = ((_d = config.colors) === null || _d === void 0 ? void 0 : _d.length) ? { color: config.colors } : {};
         const aria = {
             enabled: true,
-            decal: { show: true },
+            decal: { show: false },
             label: { enabled: true, description: config.title || config.description || "" },
         };
         let options = Object.assign(Object.assign(Object.assign(Object.assign({}, colorOpt), axis), { grid,
@@ -1117,7 +1117,7 @@ function GeoMapChart({ data, id, setEchartInstance, hFactor = 1, rowHeight, }) {
             },
             aria: {
                 enabled: true,
-                decal: { show: true },
+                decal: { show: false },
                 label: { enabled: true, description: (config === null || config === void 0 ? void 0 : config.title) || (config === null || config === void 0 ? void 0 : config.description) || "" },
             },
             tooltip,
@@ -1303,7 +1303,7 @@ function PieChart({ id, data, setEchartInstance, isMobile = false, rowHeight, hF
                 },
             } }, colorOpt), { aria: {
                 enabled: true,
-                decal: { show: true },
+                decal: { show: false },
                 label: { enabled: true, description: config.title || config.description || "" },
             }, series: Object.assign(Object.assign({}, dataSource.series), { labelLine: {
                     show: showLabels && config.labeLine,
